apps/src/lab2/views/components/Instructions/ResourcePanel/index.tsx

```diff
@@ -179,6 +179,11 @@ const ResourcePanel: React.FC<ResourcePanelProps> = ({
     }
   }, [currentTab, availableTabs]);
 
+  useEffect(() => {
+    // Reset current tab to instructions when switching levels or viewAsUserId
+    setCurrentTab(Tabs.Instructions);
+  }, [levelId, viewAsUserId]);
```

Comment:
I wonder if a teacher is viewing student work and have Version History tab open, if it would be preferable to leave the version history tab open while going clicking through students within a section. Curious what @dmcavoy thinks.

Reply:
Oh interesting. Ya a different default for teachers who are viewing student work sounds like a good follow up to explore here.